commerce/domain/orchestrators/classes/CartCalculateSampleUnitTest.cls

```diff
@@ -462,4 +490,11 @@ global class CartCalculateSampleUnitTest {
         buyerActions.setCouponChanged(True);
         return buyerActions;
     }
+
+    private static CartExtension.BuyerActionsMock getCouponChangedAtCheckoutBuyerActions(CartExtension.Cart cart) {
+        CartExtension.BuyerActionsMock buyerActions = new CartExtension.BuyerActionsMock(cart);
+        buyerActions.setCouponChanged(True);
```

Comment:
same comment as above. Can a buyer perform more than one action in a single API call? @jbisaSF